### ISSUE TITLE
Cinnamon Workspace's names reset while typing

### ISSUE THREAD

**Issue by thegreatyellow67** [linuxmint/cinnamon#11394]
Hi, today I've realized that is impossible to change workspace's names exactly where we can create and remove new workspaces (ctrl + alt + up shortcut). When I try to name one, the name will reset to whatever it currently is WHILE I'm typing. This happens in under 2 seconds (from memory, without timing it). So, in order to change the name to something non-trivial, you either have to make no mistakes and type fast, or change it piece by piece.

This is my system LMDE5 with the latest cinnamon:

System:    Kernel: 5.10.0-20-amd64 x86_64 bits: 64 compiler: gcc v: 10.2.1 Desktop: Cinnamon 5.6.5 
           tk: GTK 3.24.24 wm: muffin dm: LightDM Distro: LMDE 5 Elsie base: Debian 11.2 bullseye 
Machine:   Type: Desktop System: Gigabyte product: Z97-D3H v: N/A serial: <filter> Chassis: 
           type: 3 serial: <filter> 
           Mobo: Gigabyte model: Z97-D3H-CF v: x.x serial: <filter> UEFI: American Megatrends 
           v: F9 date: 09/18/2015 
CPU:    Info: Dual Core model: Intel Core i3-4160 bits: 64 type: MT MCP arch: Haswell rev: 3 
           L2 cache: 3 MiB 
           flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx bogomips: 28733 
           Speed: 3592 MHz min/max: 800/3600 MHz Core speeds (MHz): 1: 3592 2: 3592 3: 3592 
           4: 3592 
Graphics:  Device-1: Intel 4th Generation Core Processor Family Integrated Graphics 
           vendor: Gigabyte driver: N/A bus ID: 00:02.0 chip ID: 8086:041e 
           Device-2: NVIDIA GP108 [GeForce GT 1030] vendor: Micro-Star MSI driver: nvidia 
           v: 470.161.03 bus ID: 01:00.0 chip ID: 10de:1d01 
           Display: x11 server: X.Org 1.20.11 driver: loaded: nvidia 
           unloaded: fbdev,modesetting,vesa alternate: nouveau,nv resolution: 1920x1080~60Hz 
           s-dpi: 93 
           OpenGL: renderer: NVIDIA GeForce GT 1030/PCIe/SSE2 v: 4.6.0 NVIDIA 470.161.03 
           direct render: Yes 

At least, it's only possible to manually change the workspace's names in dconf-editor ( org / cinnamon /desktop /wm /preferences /workspace-names ).

I'm not sure what other information I need to give to fully describe the issue. Please let me know what I can provide to help.

**Comment by mtwebster** [comment]
Where are you trying to edit them? In the workspaces view? - I can't reproduce it there unfortunately.

Is editing being ended while you type? Meaning, is it behaving as though you hit enter?

**Comment by thegreatyellow67** [comment]
Yes, as described, ctrl + alt + up, standard shortcut!

**Comment by thegreatyellow67** [comment]
When I go to workspaces view ( ctrl + alt + up ) and I'm trying to edit the name of a desired workspace, while typing returned the old workspace (example: the old name was "Browsers", I want to change in "web"; so, while I try to typing "web" always returns "Browsers" each char I digit.

**Comment by thegreatyellow67** [comment]
And now, magically, works again!!!! Which type of gremlin has been installed on LMDE5, especially on newer versions of cinnamon??? :-)

**Comment by thegreatyellow67** [comment]
Mah!

**Comment by mtwebster** [comment]
Not sure - it seemed like something was hitting Escape to reset the entry each time.

Were there any warnings or errors in your ~/.xsession-errors file?

**Comment by thegreatyellow67** [comment]
The .xsession-errors content:

[xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/10352802/xsession-errors.txt)

**Comment by mtwebster** [comment]
Looks like your opacify@anish.org extension may be involved - is it up-to-date?  If it starts again, disable that and restart cinnamon. Extensions can be a bit unstable (compared to applets and desklets) as they directly modify Cinnamon code, so are more susceptible to internal changes from one Cinnamon version to the next.

**Comment by thegreatyellow67** [comment]
Yes, I've checked and seems that opacify@anish.org is the latest version. In recent times extensions or applets create many problems or don't work. Why aren't they updated? Why doesn't the Linux mint team decide to maintain them officially instead of leaving to third-party developers who then abandon the develop and don't update them? A good question! Anyway I'll try your hint. Many thanks for your support Michael.
